sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1326,7 +1326,11 @@ class Analyzer(
      * Note : In this routine, the unresolved attributes are resolved from the input plan's
      * children attributes.
      */
-    private def resolveExpressionTopDown(e: Expression, q: LogicalPlan): Expression = {
+    private def resolveExpressionTopDown(
```

Comment:
let's add comments to explain what the new parameters do

Reply:
> let's add comments to explain what the new parameters do

Done